README.md

```diff
@@ -138,8 +138,10 @@ const styles = StyleSheet.create({
 | **`headerWrapperStyle`**             | `ViewStyle`                  | Optional. Style for entire header controls wrapper. This contains the previous / next controls plus month & year.                                                                                                                                                                                                                                        |
 | **`monthTitleStyle`**                | `TextStyle`                  | Optional. Text styling for header's month text.                                                                                                                                                                                                                                                                                                          |
 | **`yearTitleStyle`**                 | `TextStyle`                  | Optional. Text styling for header's year text.                                                                                                                                                                                                                                                                                                           |
-| **`initialView`**                    | `String`                     | Optional. The view that the calendar opens to. Default is `days`. Available options are `years`, `months`, and `days`.                                                                                                                                                                                                                                   |
-| **`fontScaling`**                    | `Boolean`                    | Optional. To enable fontScaling. Default is `true`                                                                                                                                                                                                                                                                                                       |
+| **`initialView`**                    | `String`                     | Optional. The view that the calendar opens to. Default is `days`. Available options are `years`, `months`, and `days`.
+| **`renderDay`**                      | `Function`                   | Optional. Allows customization of day rendering by accepting a custom render prop that receives `{date: string; day: number, month: number, year: number, style: TextStyle}` and returns JSX Element.                                                                                                                                                                                                                                 |
```

Comment:
The `date` param is now a Date / DateFns object

Reply:
Got ya, sorry about missing that. nice catch!